landa/workspace_hooks.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+
+def validate_workspace(doc, method):
+    if doc.doctype == "Workspace":
+        if doc.hide_custom == 0:
+            doc.hide_custom = 1
+            frappe.msgprint("Die Checkbox 'Hide Custom' wurde automatisch auf 1 gesetzt.")
```

Comment:
```suggestion
	if (
		doc.for_user and
		doc.hide_custom == 0 and
		doc.extends in ("Water Body Management", "Organization Management", "Order Management")
	):
		doc.hide_custom = 1
```

Wir sollten sicherstellen, dass wir die checkbox nur nur für unsere eigenen, durch den Benutzer angepassten Workspaces setzen. In andere Apps und Standard-Workspaces wollen wir uns nicht einmischen.